.travis.yml

```diff
@@ -1,29 +1,52 @@
 language: node_js
+
 env:
 - ASCIIDOCTOR_CORE_VERSION=v1.5.5
 - ASCIIDOCTOR_CORE_VERSION=master
+
 node_js:
 - '4'
 - '6'
 - node
+
 jdk:
 - oraclejdk8
+
 script:
 - npm run lint
 - npm run package
 - npm run docs
 - SKIP_BUILD=1 npm run examples
+
 sudo: false
 
 before_deploy:
-- npm run dist
+- >
+  if ! [ "$BEFORE_DEPLOY_RUN" ]; then
+    export BEFORE_DEPLOY_RUN=1;
+    npm run dist;
+    mkdir bin
```

Comment:
1. it's possible.
2. I don't known if we have real need to reproduce this specific GH step in local.
3. If you want, you commit in my PR